### ISSUE TITLE
Implicitly set conf_files

### ISSUE THREAD

**Issue by Gottox** [void-linux/void-packages#21649]
What about marking any files in /etc automaticly as conf_files. are there any counter-examples where this would break anything?

**Comment by Duncaen** [comment]
Yes scripts in `/etc/sv/` and `/etc/runit` should probably not be `conf_files` by default, especially if a user sets `keepconf=yes` in xbps.d, which will always create `.new-$version` files even if the previous configuration file was never changed.

**Comment by sgn** [comment]
- Files provided by `grml-zsh-config`
- `/etc/bash/bashrc` (not sure about this one)

Those files shouldn't be marked as conf

**Comment by Chocimier** [comment]
Good idea.

Skimmed through xlocate, nothing besides what Duncaen pointed seems to need to be excluded.

**Comment by Gottox** [comment]
basicly Debian for example does.

**Comment by Gottox** [comment]
I'll make a PR that addresses this issue and respects @Duncaen's comments.

**Comment by travankor** [comment]
Should `/etc/sv/conf`count as conf?

**Comment by ericonr** [comment]
AppArmor profiles aren't conf files either. They are in `/etc/apparmor.d/`, but the local files in `/etc/apparmor.d/local/` are conf files.

@Duncaen what do you think about making `/etc/runit/ctrlaltdel` a `conf_file`? It is something that a user can change depending on the action they want to trigger when they use that keyboard shortcut.

> Should /etc/sv/service/confcount as conf?

@travankor IMO yes, it's where we recommend users make changes that persist across package updates. I believe the `vsv` function marks those as conf by default.

**Comment by ericonr** [comment]
Ping?

**Comment by Chocimier** [comment]
Blocked on https://github.com/void-linux/xbps/issues/342
